src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -30,7 +30,7 @@ object ScalafixPlugin extends AutoPlugin {
       inputKey[Unit](
         "Run scalafix rule(s) in this project and configuration. " +
           "For example: scalafix RemoveUnused. " +
-          "To run on test sources use test:scalafix or scalafixAll. " +
+          "To run on test sources use Test/scalafix or scalafixAll. " +
```

Comment:
Interestingly, I have always used spaces around `/` - I wonder if there is any reason to?